__tests__/__snapshots__/ord.test.js.snap

```diff
@@ -18,11 +18,11 @@ exports[`Tests for default ORD document Successfully create ORD Documents with d
       "extensible": {
         "supported": "no",
       },
-      "ordId": "capire.bookshopordsample:apiResource:undefined.AdminService:v1",
+      "ordId": "capire.bookshopordsample:apiResource:AdminService:v1",
       "partOfGroups": [
-        "sap.cds:service:capire.bookshopordsample:undefined.AdminService",
+        "sap.cds:service:capire.bookshopordsample:AdminService",
       ],
-      "partOfPackage": "capirebookshopordsample:package:undefined:v1",
+      "partOfPackage": "capirebookshopordsample:package:capire.bookshopordsample:v1",
```

Comment:
This is still wrong, it should be: 

```suggestion
      "partOfPackage": "capire.bookshopordsample:package:capirebookshopordsamplev1",
```

(not sure on the package "name" (third fragment). It could be concatnated a little bit more nicely, but that's cosmetic)

Reply:
As mentioned in the today's meeting, the package ordId is now: 
`"partOfPackage": "capire.bookshopordsample:package:customer.capirebookshopordsample:v1"`
So, @Fannon comparing to what you wrote in the last comment, the **customer.** prefix is added and **:** in front of the tag v1.
@Fannon : Looks ok now? @zongqichen ?

Reply:
> So, @Fannon comparing to what you wrote in the last comment, the customer. prefix is added and : in front of the tag v1.

Yes, `customer.` prefix ist still missing. If we have no explicit namepace set, it hsould be `customer.capirebookshopordsample:package:capirebookshopordsamplev1`